### ISSUE TITLE
node prefers mesh-on-wlan over mesh-on-vxlan

### ISSUE THREAD

**Issue by krombel** [freifunkMUC/site-ffm#57]
Useful links:
Freifunk-Franken reports sth similar: [Node on BATMAN_V cannot measure throughput and always reports 1Mb/s](
https://mantis.freifunk-franken.de/view.php?id=92)

This is currently observed on vxlan but it seems it is related to BATMAN_V support.
 
This will probably be fixed when https://github.com/freifunk-gluon/gluon/issues/1726 gets closed

**Comment by awlx** [comment]
Can be resolved by:
```
echo 100000 > /sys/class/net/vx_mesh_lan/batman_adv/throughput_override
echo 100000 > /sys/class/net/vx_mesh_wan/batman_adv/throughput_override
```

**Comment by krombel** [comment]
Forwarded upstream as it seems we are not the only ones affected: https://github.com/freifunk-gluon/gluon/issues/1728

**Comment by krombel** [comment]
This got "fixed" in 083cf05f55ce512484f4e2f904f12f231bb08a66 by using patches of FreifunkChemnitz which set the throughput of wired connections statically to a high value.

This got released in v2019.0.3

Closing it (for now) as the issue is fixed for our network although a "real fix" is outstanding.
